great_expectations/cli/checkpoint.py

```diff
@@ -67,13 +68,20 @@ def checkpoint(ctx):
     """
     ctx.obj.data_context = ctx.obj.get_data_context_from_config_file()
 
-    usage_stats_prefix = f"cli.checkpoint.{ctx.invoked_subcommand}"
+    cli_event_noun: str = "checkpoint"
```

Comment:
Should these nouns or objects also become enums? I'm okay with how they're currently laid out just thinking out loud

Reply:
Probably, but we may also be able to get these from the click library as well. Let's plan for a future refactor with that in mind.